week-1/L-functions-nested/exercise2.js

```diff
@@ -3,3 +3,15 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+function greetings(){
+    return "Hello " ;
+   
+}
+
+console.log((greetings() + mentor1).toUpperCase());
+console.log((greetings() + mentor2).toUpperCase());
+console.log((greetings() + mentor3).toUpperCase());
+console.log((greetings() + mentor4).toUpperCase());
+console.log((greetings() + mentor5).toUpperCase());
+
```

Comment:
you need to create and use a function that spells mentors name in uppercase. you do not need to use toUpperCase method inside console instead you need to create but you need to create and use nameUpperCase function.

Reply:
ok I have made the changes thank you.